pkg/minikube/problem/err_map.go

```diff
@@ -144,6 +144,12 @@ var vmProblems = map[string]match{
 		Advice: "Configure an external network switch following the official documentation, then add `--hyperv-virtual-switch=<switch-name>` to `minikube start`",
 		URL:    "https://docs.docker.com/machine/drivers/hyper-v/",
 	},
+	"HYPERV_POWERSHELL_NOT_FOUND": {
+		Regexp: re(`Powershell is not found in the path`),
```

Comment:
Thanks for making this PR, please make sure the regex, matches the exact error message,
in this case it is "Powershell was not found in the path"

Reply:
s/is/was